components/accordion/package.json

```diff
@@ -25,16 +25,12 @@
 	},
 	"main": "dist/index.css",
 	"peerDependencies": {
-		"@spectrum-css/icon": ">=7",
-		"@spectrum-css/tokens": ">=14 || >=15"
+		"@spectrum-css/icon": ">=9.0.0 <10.0.0",
+		"@spectrum-css/tokens": ">=16.0.0 <17.0.0"
```

Comment:
For a lot of these semver range changes appearing with both `>=` and `<`, is there any reason not to use the caret range instead? For example, this line could just be `"@spectrum-css/tokens": "^16.0.0`?

Reply:
They are equivalent, that's true. I just like the readability of this syntax in peerDeps; it's subtly more readable. Any reason to use caret instead?